src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7234/cache.with.freshened.response.that.updated.by.weak.validator/accept/client.rpt

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyAcceptRef ${nukleus:newRouteRef()} # external scope
+property date ${http_cache:date()} # shared property
+property date2 ${http_cache:datePlus(2)} # shared property
+property etag ${http_cache:weakEtag()}
+
+connect await ROUTED_PROXY
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+
+
+connected
+
+write close
+read nukleus:begin.ext ${http:header(":status", "200")}
+                       ${http:header("content-length", "11")}
+                       ${http:header("cache-control", "max-age=15")}
+                       ${http:header("date", date)}
+                       ${http:header("etag", etag)}
+
+read "hello world"
+
+read closed
+read notify REQUEST_CACHED
+
+# INPUT STREAM 2
+connect await REQUEST_CACHED
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+                        ${http:header("if-modified-since", date)}
+                        ${http:header("if-none-match", etag)}
+
+connected
+
+write close
+
+read nukleus:begin.ext ${http:header(":status", "304 (Not Modified)")}
```

Comment:
I would expect the 304 to be returned from the cache/nukelus and not go through.  AM I missing something.

Reply:
I was trying follow https://tools.ietf.org/html/rfc7234#section-4.3.4? Perhaps you can double confirm my understanding.

Reply:
I don't think this test is needed, (at least in this incarnation).  The cache is refreshed in the cache, BUT I would still expect a 304 to be returned the client.  I don't see anything in section-4.3.4 to change that behavior on weak etag (again, please correct me if I'm missing something).